package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "4.3.0",
+  "version": "4.3.1-alpha.0",
```

Comment:
I will bump this pre-release version with the following changes:

* export the `PrecomputedFlag` interface for setting up the memory only store's type
* update the `PRECOMPUTED_BASE_URL` and `PRECOMPUTED_FLAGS_ENDPOINT`